packages/devtools_app/lib/src/utils.dart

```diff
@@ -1305,3 +1281,18 @@ extension BoolExtension on bool {
     return -1;
   }
 }
+
+Future<T> whenValueNonNull<T>(ValueListenable<T> listenable) {
```

Comment:
I don't see where this is used in this PR?

Reply:
it is used in a future pr.